src/App.js

```diff
@@ -66,7 +64,9 @@ class App extends Component {
         }
       }
 
-    getUserMovieRatings();
+      getMoviesRequest();
```

Comment:
won't this introduce a breaking change for users who aren't logged in to see comments/movies or am I missing a small detail?